Solutions/Palo Alto - XDR (Cortex)/Playbook/azuredeploy.json

```diff
@@ -0,0 +1,463 @@
+                                                "recipient":  {
+                                                    "channelId":  "19:Ai0dUOSmL9BSgkBzZ1zXAHi3KR-XXXXHDX1UoaBy0141@thread.tacv2",

```

Comment:
ChannelId and GroupId should be taken as an input from the user and should not be hardcoded.